commands/account/clean.ts

```diff
@@ -94,22 +106,50 @@ export async function handler(
         { border: { bodyLeft: '  ' } }
       )
     );
+
+    let promptMessage = i18n(
+      oneAccountFound ? `${i18nKey}.confirm.one` : `${i18nKey}.confirm.other`,
+      {
+        count: accountsToRemove.length,
+      }
+    );
+
+    const accountOverride = getCWDAccountOverride();
+    const overrideFilePath = getDefaultAccountOverrideFilePath();
+    if (overrideFilePath && accountOverride) {
+      if (
+        accountsToRemove.some(
+          account =>
+            account.name === accountOverride ||
+            // @ts-ignore: Default account override files can only exist with global config
```

Comment:
Nit: `@ts-expect-error` has more desirable behavior than `@ts-ignore`, because when there comes a day this is no longer a ts error we will get a warning that it is unused and can be removed.  [Example](https://dev.to/maafaishal/ts-expect-error-over-ts-ignore-in-typescript-5140)